manifests/m/MarcusKleinehagenbrock/CloneSpy/3.43/MarcusKleinehagenbrock.CloneSpy.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MarcusKleinehagenbrock.CloneSpy
+PackageVersion: 3.43
+PackageLocale: en-US
+Publisher: Marcus Kleinehagenbrock

```

Comment:
```suggestion
Publisher: The CloneSpy Team
```
![image](https://user-images.githubusercontent.com/11600822/120901132-e9e8a580-c630-11eb-9d47-08d13e8db75e.png)

Reply:
Thanks for your help!

Reply:
I also updated the package identifier with this info